vumi_http_proxy/http_proxy.py

```diff
@@ -38,7 +38,6 @@ class Proxy(proxy.Proxy):
 
     def __init__(self, blacklist):
         proxy.Proxy.__init__(self)
-        self.blacklist = blacklist
```

Comment:
I removed it in a desperate attempt when I was getting an "cannot connect, already running" error, and forgot to put back